automation/run-tests.sh

```diff
@@ -27,7 +27,7 @@ PYTEST_OPTIONS="--verbose --verbose \
         --durations=5 \
         --cov /usr/lib/python*/site-packages/libnmstate \
         --cov /usr/lib/python*/site-packages/nmstatectl \
-        --cov-report=term \
+        --cov-report=xml \
```

Comment:
The term output was useful to get some direct info when running it manually or checking the travis log. You can probably add `xml` as an extra option.